pkg/deploy/dev-workspace/dev_workspace.go

```diff
@@ -55,96 +50,103 @@ var (
 	}
 )
 
-func ReconcileDevWorkspace(deployContext *deploy.DeployContext) (done bool, err error) {
+type DevWorkspaceReconciler struct {
+	deploy.Reconcilable
+}
+
+func NewDevWorkspaceReconciler() *DevWorkspaceReconciler {
+	return &DevWorkspaceReconciler{}
+}
+
+func (d *DevWorkspaceReconciler) Reconcile(ctx *deploy.DeployContext) (reconcile.Result, bool, error) {
 	if util.IsOpenShift && !util.IsOpenShift4 {
 		// OpenShift 3.x is not supported
-		return true, nil
+		return reconcile.Result{}, true, nil
 	}
 
-	if !deployContext.CheCluster.Spec.DevWorkspace.Enable {
+	if !ctx.CheCluster.Spec.DevWorkspace.Enable {
 		// Do nothing if DevWorkspace is disabled
-		return true, nil
+		return reconcile.Result{}, true, nil
 	}
 
-	if isDevWorkspaceOperatorCSVExists(deployContext) {
+	if isDevWorkspaceOperatorCSVExists(ctx) {
 		// Do nothing if DevWorkspace has been already deployed via OLM
-		return true, nil
+		return reconcile.Result{}, true, nil
 	}
 
 	if util.IsOpenShift {
-		//
-		wtoInstalled, err := doesWebTerminalSubscriptionExist(deployContext)
+		wtoInstalled, err := isWebTerminalSubscriptionExist(ctx)
 		if err != nil {
-			return false, err
+			return reconcile.Result{Requeue: true}, false, err
 		}
 		if wtoInstalled {
 			// Do nothing if WTO exists since it should bring or embeds DWO
-			return true, nil
+			return reconcile.Result{}, true, nil
 		}
 	}
 
 	if !deploy.IsDevWorkspaceEngineAllowed() {
 		// Note: When the tech-preview-stable-all-namespaces will be by default stable-all-namespaces 7.40.0?, change the channel from the log
-		exists, err := isDevWorkspaceDeploymentExists(deployContext)
+		exists, err := isDevWorkspaceDeploymentExists(ctx)
 		if err != nil {
-			return false, err
+			return reconcile.Result{Requeue: true}, false, err
 		}
 		if !exists {
 			// Don't allow to deploy a new DevWorkspace operator
-			return false, fmt.Errorf("To enable DevWorkspace engine, deploy Eclipse Che from tech-preview channel.")
+			return reconcile.Result{}, false, fmt.Errorf("To enable DevWorkspace engine, deploy Eclipse Che from tech-preview channel.")
```

Comment:
Do we need here reconcile.Result{Requeue: true} ?

Reply:
Requeue won't help so it is not required